flang/lib/Lower/Bridge.cpp

```diff
@@ -2470,6 +2524,145 @@ class FirConverter : public Fortran::lower::AbstractConverter {
     return Fortran::evaluate::GetShape(foldingContext, expr);
   }
 
+  //===--------------------------------------------------------------------===//
+  // Analysis on a nested explicit iteration space.
+  //===--------------------------------------------------------------------===//
+
+  void analyzeExplicitSpace(const Fortran::parser::ConcurrentHeader &header) {
```

Comment:
Can this be static?  It looks like all of the overloads could be static.

Reply:
No.